src/common/containers/builds.js

```diff
@@ -77,9 +81,10 @@ Builds.propTypes = {
 };
 
 const mapStateToProps = (state, ownProps) => {
-  const account = ownProps.params.account.toLowerCase();
-  const repo = ownProps.params.repo.toLowerCase();
-  const repoFullName = `${account}/${repo}`;
+  const owner = ownProps.params.owner.toLowerCase();
+  const name = ownProps.params.name.toLowerCase();
+  const fullName = `${owner}/${name}`;
+  const url = getGitHubRepoUrl(fullName);
```

Comment:
Same question as in build details https://github.com/canonical-ols/build.snapcraft.io/pull/91/files#r96604386